package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "tbk_sdk",
-  "version": "1.0.0",
+  "name": "transbank-sdk",
+  "version": "1.0.2",
```

Comment:
What happend to 1.0.1?

Reply:
Actually, it 1.0.0 never existed on `transbank-sdk`. The first version was `1.0.1`. This package is not named 'tbk_sdk' anymore.